test/test-network/conf/routing-policy-rule-test1.network

```diff
@@ -23,3 +24,9 @@ IncomingInterface=test1
 Table=9
 Priority=101
 Family=both
+
+[RoutingPolicyRule]
+IncomingInterface=test1
+From=0.0.0.0/8
+Table=10
```

Comment:
Please add `10` to `routing_policy_rule_tables` array in systemd-networkd-tests.py.

Reply:
Done